lib/actions/user.js

```diff
@@ -5,14 +5,17 @@ import { secureFetch } from '../util/middleware'
 import { isNewUser } from '../util/user'
 
 // Middleware API paths.
-const API_MONITORTRIP_PATH = '/api/secure/monitoredtrip'
+const API_MONITORED_TRIP_CHECK_SUBPATH = '/checkitinerary'
+const API_MONITORED_TRIP_PATH = '/api/secure/monitoredtrip'
 const API_OTPUSER_PATH = '/api/secure/user'
-const API_OTPUSER_VERIFYSMS_PATH = '/verify_sms'
+const API_OTPUSER_VERIFY_SMS_SUBPATH = '/verify_sms'
 
 const setCurrentUser = createAction('SET_CURRENT_USER')
 const setCurrentUserMonitoredTrips = createAction('SET_CURRENT_USER_MONITORED_TRIPS')
 const setLastPhoneSmsRequest = createAction('SET_LAST_PHONE_SMS_REQUEST')
 export const setPathBeforeSignIn = createAction('SET_PATH_BEFORE_SIGNIN')
+export const clearItineraryExistence = createAction('CLEAR_ITINERARY_AVAILABILITY')
+const setitineraryExistence = createAction('SET_ITINERARY_AVAILABILITY')
```

Comment:
nitpick: I think the action string should match the action name i.e., replace `AVAILABILITY` with `EXISTENCE`.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/242/commits/846892ab816dc32ccd6688b06042f030225c453a.